options/options.js

```diff
@@ -718,14 +718,14 @@ class OuterActionsWrapper {
 }
 
 class EngineItemWrapper {
-    constructor(val, callback) {
+    constructor(val, callback, saved) {
         this.callback = callback;
         this.onchange = this.onchange.bind(this);
 
         this.elem = document.createElement("div");
         this.elem.innerHTML = `
-            <input type="text" title="${geti18nMessage("search_name_tooltip")}"></input>
-            <input type="text" title="${geti18nMessage('search_url_tooltip')}"></input>
+            <input class="search-name-input" type="text" title="${geti18nMessage("search_name_tooltip")}"></input>
+            <input class="search-url-input" type="text" title="${geti18nMessage('search_url_tooltip')}"></input>
             <a href="#" >&#10003</a>
             <a href="#" >&#10007</a>
         `; // TODO: Create elements by JavaScript DOM.
```

Comment:
我想：使用html比较直观；使用js操作，先要创建元素，又需要给许多属性赋值
web-ext 的 lint 给出了使用innerHTML的警告:  
> Unsafe assignment to innerHTML

> Due to both security and performance concerns, this may not be set using dynamic values which have not been adequately sanitized. This can lead to security issues or fairly serious performance degradation.

那么后期修改